src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyAssignment.cs

```diff
@@ -127,6 +127,13 @@ public class NewAzurePolicyAssignmentCmdlet : PolicyCmdletBase, IDynamicParamete
         [ValidateNotNullOrEmpty]
         public Hashtable Sku { get; set; }
 
+        /// <summary>
+        /// Gets or sets the policy assignment metadata parameter
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The metadata for policy assignment. This can either be a path to a file name containing the metadata, or the metadata as string.")]
+        [ValidateNotNullOrEmpty]
+        public string Metadata { get; set; }
```

Comment:
Is there a reason this new property is not included in `Get-AzureRmPolicyAssignment`?
Additionally, what is the actual type of Metadata (meaning, how is it represented beyond a string, or is that basically user-defined)?